tools/ConsoleTypes.natvis

```diff
@@ -45,9 +45,27 @@
         </Expand>
     </Type>
 
-    <Type Name="KeyEvent">
-        <DisplayString Condition="_keyDown">{{↓ wch:{_charData} mod:{_activeModifierKeys} repeat:{_repeatCount} vk:{_virtualKeyCode} vsc:{_virtualScanCode}}</DisplayString>

```

Comment:
i did think the arrows were clever... :P

Reply:
I'll restore the arrows before this merges. 🙂